python/src/ops.cpp

```diff
@@ -3071,6 +3071,72 @@ void init_ops(nb::module_& m) {
             array: The convolved array.
       )pbdoc");
   m.def(
+      "conv3d",
+      [](const array& input,
+         const array& weight,
+         const std::variant<int, std::tuple<int, int, int>>& stride,
+         const std::variant<int, std::tuple<int, int, int>>& padding,
+         const std::variant<int, std::tuple<int, int, int>>& dilation,
+         int groups,
+         StreamOrDevice s) {
+        std::tuple<int, int, int> stride_pair{1, 1, 1};
+        std::tuple<int, int, int> padding_pair{0, 0, 0};
+        std::tuple<int, int, int> dilation_pair{1, 1, 1};
```

Comment:
The name pair here is a little confusing. Maybe just call them `strides`, `paddings`, `dilations` or something.

Reply:
Changed it.